pkg/server/list_users.go

```diff
@@ -36,6 +36,10 @@ func (s *Server) ListUsers(
 		return nil, status.Error(codes.Unimplemented, "ListUsers is not enabled. It can be enabled for experimental use by passing the `--experimentals enable-list-users` configuration option when running OpenFGA server")
 	}
 
+	if !s.IsExperimentallyEnabled(ExperimentalEnableConsistencyParams) && req.GetConsistency() == openfgav1.ConsistencyPreference_HIGHER_CONSISTENCY {
```

Comment:
Maybe we can have this in a function instead of having it repeated everywhere?

Reply:
Updated to extract a common method 👍